pkg/api/api.proto

```diff
@@ -261,6 +261,10 @@ message Environment {
       }
       repeated SyncWindow syncWindows = 1;
     }
+    message DeploymentMetaData {
+      string deployAuthor = 1;
+      string deployTime = 2;
```

Comment:
```suggestion
      // we use a string here, because the UI cannot handle int64 as a type.
      // the string contains the unix timestamps in seconds (utc)
      string deployTime = 2;
```